### PR TITLE
feat(pkger): adds xy chart

### PR DESCRIPTION
**Pull request by dearyhud**
Visual of XY chart added via the pkger CLI
Screen Shot 2019-11-05 at 11 41 21 AM

Describe your proposed changes here.

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: pkger/parser.go [hunk 1]
```diff
@@ -575,6 +575,8 @@ func parseChart(r Resource) (chart, []failure) {
 		YPos:        r.intShort("yPos"),
 		Height:      r.intShort("height"),
 		Width:       r.intShort("width"),
+		Geom:        r.stringShort("geom"),
+		ShadeBelow:  r.boolShort("shadeBelow"),
```
Comment (jsteenb2) [inline pkger/parser.go]: does this already exist above on line:
https://github.com/influxdata/influxdb/pull/15767/files#diff-bee826069937abe74ce78a93cc101d2fR571

if so then axe this

File: ui/yarn.lock [hunk 2]
```diff
@@ -1,12131 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (jsteenb2) [inline ui/yarn.lock]: why is this?

File: pkger/parser_test.go [hunk 3]
```diff
@@ -1192,6 +1192,227 @@ spec:
...
+
+		t.Run("handles invalid config", func(t *testing.T) {
+			tests := []struct {
+				name      string
+				jsonStr   string
+				numErrs   int
+				errFields []string
+			}{
+				{
+					name:      "color missing hex value",
```
Comment (jsteenb2) [inline pkger/parser_test.go]: lot of missing test cases here it looks like, don't think its a deal breaker here, but lets be cognizant of this moving forward. probably behooves us to put together some helpers to validate these validation errors across multiple view types

Comment (dearyhud) [reply pkger/parser_test.go]: totally agree. feels like we should do that sooner than later. the in-lined json/yml is making this file harder to scroll through, and likely will get harder w/ the next set of charts

**Pull request merged**